imap_processing/tests/idex/test_idex_l2b.py

```diff
@@ -5,26 +5,43 @@
 import xarray as xr
 from numpy.testing import assert_array_equal
 
-from imap_processing.cdf.utils import write_cdf
+from imap_processing.cdf.utils import load_cdf, write_cdf
+from imap_processing.idex.idex_constants import FG_TO_KG, SECONDS_IN_DAY
 from imap_processing.idex.idex_l2b import (
+    CHARGE_BIN_EDGES,
+    MASS_BIN_EDGES,
+    SPIN_PHASE_BIN_EDGES,
+    bin_spin_phases,
+    compute_counts_by_charge_and_mass,
+    compute_rates_by_charge_and_mass,
+    get_science_acquisition_on_percentage,
     get_science_acquisition_timestamps,
     idex_l2b,
-    round_spin_phases,
 )
+from imap_processing.tests.idex.conftest import L1B_EVT_CDF
+
+ONE_DAY_NS = 86400000000000
```

Comment:
Move this into the other constants area and call it NANOSECOND_IN_DAY to match the naming convention there. You should be able to do the math for it too: `NANOSECONDS_IN_DAY = SECONDS_IN_DAY * int(1e9)`